src/main/common/components/free-mediation/freeMediationTask.ts

```diff
@@ -11,11 +12,20 @@ export class FreeMediationTask {
         mediationDraft.willYouTryMediation && mediationDraft.willYouTryMediation.option === FreeMediationOption.YES)
   }
 
+  static isEnhancedWillYouTryMediationCompleted (mediationDraft: MediationDraft): boolean {
+    return (mediationDraft.willYouTryMediation && (mediationDraft.willYouTryMediation.option === FreeMediationOption.NO
+      || mediationDraft.willYouTryMediation.option === FreeMediationOption.YES))
```

Comment:
is there any other optionwhich comes as part of "willYouTryMediation"? why do we need to check is it yes or no?